cabal-install/Distribution/Solver/Types/PackageIndex.hs

```diff
@@ -210,10 +210,10 @@ deletePackageName name =
   delete name (\pkg -> packageName pkg == name)
 
 -- | Removes all packages satisfying this dependency from the index.
---
+-- Does not take into account sublibraries, only version ranges.
```

Comment:
Err... so does it *not* delete sublibraries or does it delete all sublibraries in the package? I guess the answer to this question depends on if the `name` index here is munged or not.

Reply:
Yeah, I didn't think of that. This function isn't even used anywhere though, and cabal-install's api isn't consiedered stable